pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -63,10 +63,16 @@ func SetDefaults_KubeProxyConfiguration(obj *KubeProxyConfiguration) {
 		obj.BindAddress = "0.0.0.0"
 	}
 	if obj.HealthzPort == 0 {
-		obj.HealthzPort = 10249
+		obj.HealthzPort = 10256
 	}
 	if obj.HealthzBindAddress == "" {
-		obj.HealthzBindAddress = "127.0.0.1"
+		obj.HealthzBindAddress = "0.0.0.0"
+	}
+	if obj.MetricsPort == 0 {
+		obj.MetricsPort = 10249
```

Comment:
10249 is what it used to be, I think it would be better to keep it (some e2e tests rely on it)?